docs/whatsnew-4.4.rst

```diff
@@ -0,0 +1,531 @@
+(◕ᴥ◕) <ratson@users.noreply.github.com>
+Robert Kopaczewski <rk@23doors.com>
+Samuel Huang <samhuang91@gmail.com>
+Sebastian Wojciechowski <42519683+sebwoj@users.noreply.github.com>
+Seunghun Lee <waydi1@gmail.com>
+Shanavas M <shanavas.m2@gmail.com>
+Simon Charette <charettes@users.noreply.github.com>
+Simon Schmidt <schmidt.simon@gmail.com>
+srafehi <shadyrafehi@gmail.com>
+Steven Sklar <sklarsa@gmail.com>
+Tom Booth <thomasbo@microsoft.com>
+Tom Clancy <ClancyTJD@users.noreply.github.com>
+Toni Ruža <gmr.gaf@gmail.com>
+tothegump <tothegump@gmail.com>
+Victor Mireyev <victor@opennodecloud.com>
+Vikas Prasad <vikasprasad.prasad@gmail.com>
+walterqian <walter@color.com>
+Willem <himself@willemthiart.com>
+Xiaodong <xd_deng@hotmail.com>
+yywing <386542536@qq.com>
+
+.. note::
+
+    This wall was automatically generated from git history,
+    so sadly it doesn't not include the people who help with more important
+    things like answering mailing-list questions.
+
+
+Upgrading from Celery 4.3
+=========================
+
+Please read the important notes below as there are several breaking changes.
+
+.. _v430-important:
+
+Important Notes
+===============
+
+Supported Python Versions
+-------------------------
+
+The supported Python Versions are:
+
+- CPython 2.7
+- CPython 3.5
+- CPython 3.6
+- CPython 3.7
+- CPython 3.8
+- PyPy2.7 7.2 (``pypy2``)
+- PyPy3.5 7.1 (``pypy3``)
+- PyPy3.6 7.2 (``pypy3``)
+
+Kombu
+-----
+
+Starting from this release, the minimum required version is Kombu 4.6S.
+
+
+Billiard
+--------
+
+Starting from this release, the minimum required version is Billiard 3.6.1.
+
+Eventlet Workers Pool
+---------------------
+
+We now require `eventlet>=0.24.1`.
+
+If you are using the eventlet workers pool please install Celery using:
+
+.. code-block:: console
+
+  $ pip install -U celery[eventlet]
+
+MessagePack Serializer
+----------------------
+
+We've been using the deprecated `msgpack-python` package for a while.
+This is now fixed as we depend on the `msgpack` instead.
+
+If you are currently using the MessagePack serializer please uninstall the
+previous package and reinstall the new one using:
+
+.. code-block:: console
+
+  $ pip uninstall msgpack-python -y
+  $ pip install -U celery[msgpack]
+
+MongoDB Result Backend
+-----------------------
+
+We now support the `DNS seedlist connection format <https://docs.mongodb.com/manual/reference/connection-string/#dns-seedlist-connection-format>`_ for the MongoDB result backend.
+
+This requires the `dnspython` package.
+
+If you are using the MongoDB result backend please install Celery using:
+
+.. code-block:: console
+
+  $ pip install -U celery[mongodb]
+
+Redis Message Broker
+--------------------
+
+Due to multiple bugs in earlier versions of py-redis that were causing
+issues for Celery, we were forced to bump the minimum required version to 3.3.0.
+
+Redis Result Backend
+--------------------
+
+Due to multiple bugs in earlier versions of py-redis that were causing
```

Comment:
We should probably mention the healthcheck changes and alert users that they should configure it.